### DISCUSSION TITLE
Very Confused

### DISCUSSION THREAD

**Discussion by ghost** [kibook/pmms#40] [Q&A]
Hello,

I have the script added to my server, How do i add media players to the map?

Running GTAV ESX Legacy

**Comment by TayMcKenzieNZ** [comment]
Hello, so this script is standalone. 

You need to execute the permissions provided by this script, in your server.cfg.

Once permissions are set up, you can use it with existing TV screens, radios, vehicles, around the map by typing /pmms near them and selecting the correct one, and entering a URL.

If you want to add TVs and radios to locations on the map, you'll need to use something like Menyoo, Kibook's spooner script, or Codewalker if you want it there permanently.

For codewalker, if you want to add props to the outside world, they need to be added to the ymap, if you want to add props to interiors (ie Franklin's Mansion or a custom interior), they'd need to be added to the MLO's ytyp, in the correct room.

You can grab codewalker from their discord and get support for using it here:

https://discord.gg/MKzzKKxFv8
